### PR TITLE
Add fuzzing for animation encoding then decoding.

### PR DESCRIPTION
**Pull request by maryla-uc**
Update version in aom.cmd to avoid a bug for high bit depth images.

### CODE REVIEW

File: tests/gtest/avif_fuzztest_enc_dec_anim.cc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+constexpr int kMaxFrames = 10;
+
+FUZZ_TEST(EncodeDecodeAvifFuzzTest, EncodeDecodeAnimation)
+    .WithDomains(ArbitraryAvifImage(),
+                 fuzztest::VectorOf(
+                     fuzztest::StructOf<FrameOptions>(
+                         /*duration=*/fuzztest::InRange(1, 10),
+                         fuzztest::BitFlagCombinationOf<avifAddImageFlags>(
+                             {AVIF_ADD_IMAGE_FLAG_FORCE_KEYFRAME})))
+                     .WithMinSize(2)
```
Comment (y-guyon) [inline tests/gtest/avif_fuzztest_enc_dec_anim.cc:92]: Why not `.WithMinSize(1)`?

File: tests/gtest/avif_fuzztest_enc_dec_anim.cc [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+constexpr int kMaxFrames = 10;
+
+FUZZ_TEST(EncodeDecodeAvifFuzzTest, EncodeDecodeAnimation)
+    .WithDomains(ArbitraryAvifImage(),
+                 fuzztest::VectorOf(
+                     fuzztest::StructOf<FrameOptions>(
+                         /*duration=*/fuzztest::InRange(1, 10),
+                         fuzztest::BitFlagCombinationOf<avifAddImageFlags>(
+                             {AVIF_ADD_IMAGE_FLAG_FORCE_KEYFRAME})))
```
Comment (y-guyon) [inline tests/gtest/avif_fuzztest_enc_dec_anim.cc:91]: Why not `AVIF_ADD_IMAGE_FLAG_NONE` too?

**Review by y-guyon — APPROVED** [review]

File: tests/gtest/avif_fuzztest_enc_dec_anim.cc (hunk 1 above)
Comment (maryla-uc) [reply tests/gtest/avif_fuzztest_enc_dec_anim.cc]: I thought 1 frame was not really an animation but I guess it doesn't hurt to test it.

File: tests/gtest/avif_fuzztest_enc_dec_anim.cc (hunk 2 above)
Comment (maryla-uc) [reply tests/gtest/avif_fuzztest_enc_dec_anim.cc:93]: Added a comment. BitFlagCombinationOf() does not accept a flag whose value is 0, but 0 is always part of the set anyway.

**Pull request merged**